app/src/lib/registry.ts

```diff
@@ -85,3 +93,32 @@ export function readRegistryKeySafe(
     })
   })
 }
+
+/**
+ * Read registry keys found at the expected location.
+ *
+ * This method will return an empty list if the expected key does not exist,
+ * instead of throwing an error.
+ *
+ * @param key The registry key to lookup
+ */
+export async function readRegistryKeySafe(
+  key: string
+): Promise<ReadonlyArray<IRegistryEntry>> {
+  const exists = await pathExists(batchFilePath)
+  if (!exists) {
+    log.error(
+      `Unable to find batch script at expected location: '${batchFilePath}'`
+    )
+    return []
+  }
+
+  const activeCodePage = await getActiveCodePage()
+  if (activeCodePage) {
```

Comment:
This branch is empty, can we remove it or was something accidentally?

Reply:
Accidental. Gonna :fire: it.